### PR TITLE
Update deployment docs

### PR DESCRIPTION
**Pull request by ArtFlag**
Add a comment about editing the `branch.only` property to match the tag names.

### CODE REVIEW

File: user/deployment/releases.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Or, if you're using a private repository:
...
 
+If you are using the [`branches.only` property](/user/customizing-the-build#Building-Specific-Branches), remember that when you push a tag, the [`$TRAVIS_BRANCH` property](/user/environment-variables/#Default-Environment-Variables) contains the name of the tag. As a result, edit the `branches.only` property to add the names of the tags you might push in the future. You can use a regular expression if you have formalized names. For example: 
+
+```yaml
+   branches:
+    only:
+    - master
+    - /^v.*$/
+```
+
```
Comment (plaindocs) [inline user/deployment/releases.md]: Thanks for this. It is probably worth explaining the intent of the regular expression for people who aren't familiar with them?

Comment (ArtFlag) [reply user/deployment/releases.md]: Sure. I added a quick explanation. I imagine that people tinkering with Travis already know about regexp :)

**Comment by ArtFlag** [comment]
Any chance to merge this soonish?

File: user/deployment/releases.md (hunk 1 above)
Comment (plaindocs) [reply user/deployment/releases.md]: You might be surprised. I think the new explanation suffices though - thank you!

**Pull request merged**